src/Indice.Features.Risk.Core/Data/Models/RiskEvent.cs

```diff
@@ -17,4 +17,6 @@ public class RiskEvent
     public string? Name { get; set; }
     /// <summary>The type of the event.</summary>
     public string Type { get; set; } = string.Empty;
+    /// <summary>The data of the event.</summary>
+    public dynamic? Data { get; set; }
```

Comment:
προτείνω τα εξής, να τα βλέπει και πλατφόρμα. 

1. Να έχουμε ενα System/Client πεδίο, το οποιο ξεχωρίζει το σύστημα που έστειλε το ρισκο. Τώρα είναι μονο ebanking api αλλά άμα αυριο έχεις πολλά συστήματα που θες να τροφοδοτήσουν πως θα ρωτάς για αυτό που σε νοιαζει. 
2. Να αποκτήσουμε και ενα CustomerId γιατί μπορεί να ειναι σα το case, να μην ειναι πραγματικά logged in ο πελάτης αλλα να τον ξέρει κάποιο σύστημα τρίτο που χειριζεται το auth.